SurfaceTopography/Container/ScaleDependentStatistics.py

```diff
@@ -94,9 +96,16 @@ def scale_dependent_statistical_property(container, func, n, distance, unit):
     for topography in container:
         topography = topography.to_unit(unit)
 
-        # Only compute the statistical property for distances that actually exist for this specific topography
-        l, u = topography.bandwidth()
-        m = np.logical_and(distance > l, distance < u)
+        # Only compute the statistical property for distances that actually exist for this specific topography...
+        L, u = topography.bandwidth()
```

Comment:
why did you rename `l` to `L`? `l, u` stands for lower and upper, but `L` usually stands for the scan size that is the upper end of the bandwidth.

I think we should use explicit names like `short` and `long`

Reply:
Renamed this to `left` and `right`. The reason I switched to uppercase is that flake8 complains about variables named small `L`, because in some fonts that is indistinguishable from the number one. But you are of course right about being more explicit...